### PR TITLE
Homework week 1 Zekai Koc

### PR DESCRIPTION
**Pull request by Zekai-Koc**
Hi. I have multiple github accounts and i forked the repo with a different account named "Michael".  that is my alias.

### CODE REVIEW

File: 3-UsingAPIs/Week1/homework/ex1-johnWho.js [hunk 1]
```diff
@@ -8,22 +8,28 @@ Rewrite this function, but replace the callback syntax with the Promise syntax:
...
-  setTimeout(() => {
-    if (!firstName) {
-      callback(new Error("You didn't pass in a first name!"));
-      return;
-    }
 
-    const fullName = `${firstName} Doe`;
+const getAnonName = (firstName) => {
+  return new Promise((resolve, reject) => {
+    try {
```
Comment (chrisdoc) [inline 3-UsingAPIs/Week1/homework/ex1-johnWho.js:14]: @Zekai-Koc this `try {...} catch(e) {...}` block is needed here

File: 3-UsingAPIs/Week1/homework/ex2-checkDoubleDigits.js [hunk 2]
```diff
@@ -12,14 +12,30 @@ Complete the function called `checkDoubleDigits` such that:
   "Expected a double digit number but got `number`", where `number` is the 
   number that was passed as an argument.
 ------------------------------------------------------------------------------*/
-function checkDoubleDigits(/* TODO add parameter(s) here */) {
-  // TODO complete this function
+function checkDoubleDigits(number) {
+  return new Promise((resolve, reject) => {
+    try {
```
Comment (chrisdoc) [inline 3-UsingAPIs/Week1/homework/ex2-checkDoubleDigits.js:17]: Again this `try {...} catch(e) {...} block` is needed here as there is nothing that could throw an exception inside

**Review by chrisdoc — CHANGES_REQUESTED** [review]
Two minor things regarding the excess try catch

File: 3-UsingAPIs/Week1/homework/ex2-checkDoubleDigits.js (hunk 2 above)
Comment (Zekai-Koc) [reply 3-UsingAPIs/Week1/homework/ex2-checkDoubleDigits.js:17]: as a best practice i prefer try/catch blocks. 
btw, i had them in the other exercises as well but tester made me delete them all. :)
should i delete or keep?

File: 3-UsingAPIs/Week1/homework/ex1-johnWho.js (hunk 1 above)
Comment (Zekai-Koc) [reply 3-UsingAPIs/Week1/homework/ex1-johnWho.js:14]: as a best practice i prefer try/catch blocks. 
btw, i had them in the other exercises as well but tester made me delete them all. :)
should i delete or keep?

Comment (chrisdoc) [reply 3-UsingAPIs/Week1/homework/ex1-johnWho.js:14]: If the code doesn't throw any exceptions then adding additional try / catch does not have any purpose, hence I would remove them as they don't do anything in this case :)

**Review by chrisdoc — APPROVED** [review]

**Comment by chrisdoc** [comment]
LGTM 🚀 Great work

**Pull request closed without merging**